.gitignore

```diff
@@ -0,0 +1 @@
+node_modules/
```

Comment:
Can you please find a  better .gitignore templates for projects like this please?